test/sharness/t0042-add-skip.sh

```diff
@@ -48,6 +48,17 @@ test_add_skip() {
 		test_cmp expected actual
 	'
 
+	test_expect_success "'ipfs add' includes hidden files at the top-level even without --hidden" '
```

Comment:
maybe say `'ipfs add' includes hidden files given explicitly even without --hidden` -- top-level was a bit confusing given the example below (`mountdir/dotfiles/.vimrc`), took me a second to realize it meant "top level of the commands file tree"